src/util/server.c

```diff
@@ -472,6 +474,12 @@ int server_setup(const char *name, int flags,
     char *pidfile_name;
     int cfg_debug_level = SSSDBG_INVALID;
 
+    if (is_responder) {
```

Comment:
This won't compile as `is_responder` arg added in a later commit...

Reply:
Fixed.